crates/docs/src/validation.rs

```diff
@@ -23,7 +23,16 @@ pub fn extract_snippets_from_file(
                 .name("config")
                 .map_or_else(String::new, |m| m.as_str().to_string());
             let command_match = caps.name("command")?;
-            let output = caps.name("output").map(|m| m.as_str().to_string());
+            let output = caps.name("output").map(|m| {
+                let gas_re = Regex::new(r"gas: ~\d+").unwrap();
+                let execution_resources_re =
+                    Regex::new(r"(steps|memory holes|builtins|syscalls): (\d+|\(.+\))").unwrap();
+
+                let output = gas_re.replace_all(m.as_str(), "gas: ~[..]").to_string();
+                execution_resources_re
+                    .replace_all(output.as_str(), "${1}: [..]")
+                    .to_string()
+            });
```

Comment:
Let's optimize regex here, we can use `Lazy` or `LazyLock`
https://docs.rs/regex/latest/regex/#avoid-re-compiling-regexes-especially-in-a-loop

```rust
static RE_EXAMPLE: LazyLock<Regex> = LazyLock::new(|| {
    Regex::new( r"regex here").unwrap()
});
```